### DISCUSSION TITLE
Multiple selection with checkboxes

### DISCUSSION THREAD

**Discussion by Lasoty** [Megabit/Blazorise#3397] [Ideas]
A very useful component would be DropDown with a checkbox list.

![image](https://user-images.githubusercontent.com/23433922/152559234-23c4ffb9-23ee-4110-9999-499640aab805.png)

**Comment by David-Moreira** [comment]
Our Autocomplete with 
- multiple selection (#2984)
- configurable inner content RenderFragment (#3012)
-  #3302

is one step away from this, right @stsrki?

**Reply by stsrki** [reply]
Yes, feature-wide we're already there. The only thing missing are the check boxes. And we can do that on v1.1.
